src/main/java/uk/ac/ebi/ena/webin/cli/WebinCliExecutor.java

```diff
@@ -149,10 +151,9 @@ private void checkGenomeSubmissionRatelimit() {
             RateLimitResult ratelimit;
             try {
                 RatelimitService ratelimitService = new RatelimitService.Builder()
-                        .setCredentials(getParameters().getWebinServiceUserName(),
-                                getParameters().getPassword())
-                        .setTest(getParameters().isTest())
-                        .build();
+                    .setWebinRestUri(UrlUtils.getWebinRestUrl(getParameters().isTest()))
```

Comment:
- Maybe we could replace UrlUtils + statics with UrlProvider instance that is configured with isTest and returns the various URL.